helm-chart/kuberay-operator/templates/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.defaultRayEnvs }}
```

Comment:
This check could be a problem if we want to add more fields to the configuration. Is it possible that we always generate the configmap?